### PR TITLE
BUGZ-187: Proper Render Scripting Interface with renderMethod property

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.atlassian.net/browse/BUGZ-187

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19495/HighFidelity-Beta-PR15569-a139766.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19495/HighFidelity-Quest-Beta-PR15569-a139766.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19495/HighFidelity-Beta-PR15569-a139766.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19495/HighFidelity-Beta-Interface-PR15569-a139766.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19495/HighFidelity-Beta-PR15569-a139766.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19495/HighFidelity-Beta-Interface-PR15569-a139766.exe

File: interface/src/Menu.cpp [hunk 1]
```diff
@@ -522,6 +523,24 @@ Menu::Menu() {
     addCheckableActionToQMenuAndActionHash(renderOptionsMenu, MenuOption::ComputeBlendshapes, 0, true,
         DependencyManager::get<ModelBlender>().data(), SLOT(setComputeBlendshapes(bool)));
 
+#if defined(Q_OS_MAC) || defined(Q_OS_WIN)
```
Comment (SamGondelman) [inline interface/src/Menu.cpp]: I have to ifdef this out on linux or else I get an unused variable warning....I don't know why

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19498/HighFidelity-Beta-PR15569-0d6d460.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19498/HighFidelity-Beta-Interface-PR15569-0d6d460.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19498/HighFidelity-Beta-PR15569-0d6d460.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19498/HighFidelity-Beta-Interface-PR15569-0d6d460.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19498/HighFidelity-Beta-PR15569-0d6d460.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19498/HighFidelity-Quest-Beta-PR15569-0d6d460.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19505/HighFidelity-Beta-PR15569-5a65e95.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19505/HighFidelity-Quest-Beta-PR15569-5a65e95.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19505/HighFidelity-Beta-PR15569-5a65e95.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19505/HighFidelity-Beta-Interface-PR15569-5a65e95.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19505/HighFidelity-Beta-PR15569-5a65e95.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19505/HighFidelity-Beta-Interface-PR15569-5a65e95.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19513/HighFidelity-Beta-PR15569-bbdcaa8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19513/HighFidelity-Beta-Interface-PR15569-bbdcaa8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19513/HighFidelity-Beta-PR15569-bbdcaa8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19513/HighFidelity-Quest-Beta-PR15569-bbdcaa8.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19513/HighFidelity-Beta-PR15569-bbdcaa8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19513/HighFidelity-Beta-Interface-PR15569-bbdcaa8.exe

**Comment by SamGondelman** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19524/HighFidelity-Beta-PR15569-db813fa.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19524/HighFidelity-Quest-Beta-PR15569-db813fa.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19524/HighFidelity-Beta-PR15569-db813fa.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19524/HighFidelity-Beta-Interface-PR15569-db813fa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19524/HighFidelity-Beta-PR15569-db813fa.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19524/HighFidelity-Beta-Interface-PR15569-db813fa.exe

**Pull request merged**